apps/web/src/app/arrangementer/components/mutations.ts

```diff
@@ -15,11 +17,13 @@ export const useDeregisterMutation = () => {
           return
         }
 
-        await Promise.all([
-          await queryClient.invalidateQueries(
-            trpc.event.attendance.getAttendance.queryOptions({ id: input.attendanceId })
-          ),
-        ])
+        if (trpcSSERegisterChangeConnectionState !== "pending") {
```

Comment:
Maybe a comment saying that `pending` means the connection is alive, and that we only invalidate since the connection can't provide the fresh data? Otherwise I think this could confuse people as to why it is here